packages/react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -149,7 +149,7 @@ const _getMenuItemStylesFunction = memoizeFunction(
     ...styles: (IStyleFunctionOrObject<IContextualMenuItemStyleProps, IContextualMenuItemStyles> | undefined)[]
   ): IStyleFunctionOrObject<IContextualMenuItemStyleProps, IContextualMenuItemStyles> => {
     return (styleProps: IContextualMenuItemStyleProps) =>
-      concatStyleSetsWithProps(styleProps, getItemStyles, ...styles);
+      concatStyleSetsWithProps(styleProps, getItemStyles, ...styles) as IContextualMenuItemStyles;
```

Comment:
after bump gives a pretty huge error, expects only DeepPartial<...> but because of `concatStyleWithProps` infers to DeepPartial<DeepPartial<...> 
<img width="1903" alt="Screenshot 2024-11-12 at 13 11 18" src="https://github.com/user-attachments/assets/e17c8424-5b92-4843-b6ef-3a26e50308fb">